cypress/e2e/3-testing/invalidLogin.cy.js

```diff
@@ -0,0 +1,17 @@
+describe("invalid login", () => {
+  it("visits site and inputs incorrct credentials", () => {
+    cy.visit("https://bjedne.github.io/social-media-client/");
```

Comment:
```suggestion
    cy.visit("/");
```

You should be testing the code in the current branch (dev code) not production code that has already been deployed. Cypress runs it's own server, so you can just pass the relative path.